Tests/test_imagecms.py

```diff
@@ -209,13 +209,13 @@ def test_exceptions() -> None:
         ImageCms.buildTransform("foo", "bar", "RGB", "RGB")
 
     with pytest.raises(ImageCms.PyCMSError, match="Invalid type for Profile"):
-        ImageCms.getProfileName(None)
+        ImageCms.getProfileName(None)  # type: ignore[arg-type]
```

Comment:
What does this warning look like?

Reply:
```
Tests/test_imagecms.py:212: error: Argument 1 to "getProfileName" has incompatible type "None"; expected
"Union[str, SupportsRead[bytes], CmsProfile, ImageCmsProfile]"  [arg-type]
            ImageCms.getProfileName(None)
                                    ^~~~
```
The test's intention is to go through
https://github.com/python-pillow/Pillow/blob/c7574392b050bf93334ec3c1b1873de2cc3487fb/src/PIL/ImageCms.py#L807
where
https://github.com/python-pillow/Pillow/blob/c7574392b050bf93334ec3c1b1873de2cc3487fb/src/PIL/ImageCms.py#L369-L371
and then
https://github.com/python-pillow/Pillow/blob/c7574392b050bf93334ec3c1b1873de2cc3487fb/src/PIL/ImageCms.py#L239-L240
to reach
https://github.com/python-pillow/Pillow/blob/c7574392b050bf93334ec3c1b1873de2cc3487fb/src/PIL/ImageCms.py#L263
The type is deliberately incorrect.

Reply:
Thanks!